pyintelowl/cli/analyse.py

```diff
@@ -145,3 +145,34 @@ def file(
         )
     except IntelOwlClientException as e:
         ctx.obj.logger.fatal(str(e))
+
+
+@analyse.command(help="Send multiple analysis requests from file")
+@click.argument("filepath", type=click.Path(exists=True, resolve_path=True))
+@click.pass_context
+def batch(
+    ctx: ClickContext,
+    filepath: str,
+):
+    data = get_json_data(filepath)
+    for el in data:
```

Comment:
extract the for loop into another function which should be present under `IntelOwl class`